File: src/array/array_java/Ch2_SearchingInArray.java
package array.array_java;

import java.util.Scanner;

public class Ch2_SearchingInArray {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();
        int[] arr = new int[n];

        for (int i = 0; i < arr.length; i++) {
            arr[i] = scn.nextInt();
        }

        int data = scn.nextInt();

        int index = -1;
        for (int i = 0; i < arr.length; i++) {
                if (arr[i]==data){
                    index =i;
                    break;
                }
        }

        System.out.println(index);
    }
}

File: src/array/array_kotlin/Ch2_SearchingInArray.kt
package array.array_kotlin

import java.util.*

fun main() {
    searchingInArray()
}

fun searchingInArray() {
    val scn = Scanner(System.`in`)
    val n = scn.nextInt()

    val arr = IntArray(n)

    for (i in arr.indices) {
        arr[i] = scn.nextInt()
    }

    val data = scn.nextInt()

    var index = -1
    for (i in arr.indices) {
        if (arr[i] == data) {
            index = i
            break
        }
    }

    println(index)
}
